docs/arch/adr-27.md

```diff
@@ -0,0 +1,53 @@
+# ADR 27: Subject Viewer Config
+
+Created: June 26, 2020
+
+## Context
+
+A workflow determines which subject viewer to use with the `workflow.configuration.subject_viewer` property. Subject viewers include `singleImage`, `lightCurve`, `multiFrame`, and `subjectGroup` at creation of this ADR. Some subject viewers can utilize, or require, additional configuration information.
+
+### [multiFrame](https://github.com/zooniverse/front-end-monorepo/tree/master/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/MultiFrameViewer)
+
+A workflow using the multi-frame subject viewer might have a preference regarding:
+marks per frame: some might prefer marks filtered per frame, like a transcription workflow where each frame represents a unique page to transcribe with marks only relevant to each page, while other workflows might prefer marks persist between frames [, like ?] (`multi_image_clone_markers` in PFE)
```

Comment:
Yes, I think this is currently `multi_image_clone_markers` in PFE, however, this setting is slightly confusing to some project builders. I've had some ask if this setting creates new cloned marks of the original mark onto the other frames, which would add marks into the annotation. I've had to explain that this setting is just for visibility between frames of existing marks on other frames and doesn't auto-create additional marks in the annotation. We should note that confusion about this setting here.

Reply:
It was originally requested by Space Warps, and they might be the only project that have used it.

Reply:
We should check on that because it's not behind a feature flag and the checkbox is available to everyone in the lab UI. As I said, I've directly responded to questions in tickets about it.

Reply:
Yeah, if we could get a list of projects that are using it, that would be useful. I remember, from testing on Space Warps, that it didn't work well with subtask popups either.

Reply:
Here's a list of launch approved projects with active workflows using `multi_image_clone_markers`:

Agent NEO
Brain Match
Eye for Diabetes
Fishing in the past
ForestEyes
Galaxy Builder
Galaxy Zoo: Clump Scout
Heritage Quest
Hubble Asteroid Hunter
Local Group Cluster Search
Michigan ZoomIN
Monkey Health Explorer
Planetary Response Network & Rescue Global: Caribbean Storms 2017
Poppin' Galaxy
Power to the People
Project Plumage
Radio Galaxy Zoo: LOFAR
Run, Herring, Run!
Snapshot Camdeboo
South Sudan DiversityCam
Space Warps - HSC

I spot-checked 5 - 2 didn't have multi-image subjects for the subject I saw (Fishing in the Past, Hubble Asteroid Hunter), 2 using survey task (Michigan ZoomIN, South Sudan Diversity Cam, which persists the tag choice between frames regardless of `multi_image_clone_markers`), and 1 used as intended (Power to the People), but this is just a rough spot-check, not determinate.

So it's definitely being used, but it's not clear to the project builders what the setting is doing.

Reply:
Let's add some of this info into the ADR here. It's good to have confirmation that project builders don't understand this setting.